src/index.ts

```diff
@@ -2,32 +2,14 @@ import fastifyFormBody from '@fastify/formbody';
 import fastify from 'fastify';
 import jwt from 'jsonwebtoken';
 
+import Config from './oauth-mock-server.json';
```

Comment:
@hmathavan Importing wont work here. By importing the config would be compiled into the code when building the app. Instead you need to load the file content on demand. Could you place the logic for loading the config file in a file called `config.ts`. In the `index.ts` file you should then be able to load data by this `const config = getConfig();` and use `config.users` instead of `users`.

```ts
import fs from 'fs';
import { cwd } from 'node:process';
import path from 'path';

function getConfig() {
const defaultConfig = {
  realm: 'my-project',
  users: [
  {
    id: '1',
    username: 'toni',
    email: 'toni@test.com',
    name: 'Toni Tester',
  },
  {
    id: '2',
    username: 'alice',
    email: 'alice@wonderland.org',
    name: 'Alice Wonderland',
  },
  {
    id: '3',
    username: 'herbert',
    email: 'her@bert.de',
    name: 'Herbert',
  },
  ]
};

const configPath = path.join(cwd, 'oauth-mock-server.json');
if (!fs.existsSync(configPath)) {
  return defaultConfig;
}

const rawConfig = fs.readFileSync('student.json');
const config = {};
Object.assign(config, defaultConfig, JSON.parse(rawdata));
return config;
}
```